inspect_byproducts.py

```diff
@@ -172,5 +184,6 @@ def main():
       inspect_byproducts(args.link, args.outerr, args.operator, args.string)
 
 
+
```

Comment:
Alright, I did it using 'sys.exit(<return code>)'. Shall I go for 'ArgumentParser.error(<message>)' ?